pkg/storaged/block/format-dialog.jsx

```diff
@@ -587,6 +610,17 @@
                         return client.blocks_fsys[path];
                 }
 
+                function block_swap_for_block(path) {
+                    if (keep_keys) {
+                        const content_block = client.blocks_cleartext[path];
+                        return client.blocks_swap[content_block.path];
+                    } else if (is_encrypted(vals))
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19843-20240117-084217-6aab0478-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)